nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -123,7 +122,6 @@ def _apply(
         dataset: Optional[Dataset] = None,
     ) -> TModel:
         self._set_backend_entity(model)
-        model = self._backend_entity.insert_null_biases(model)
```

Comment:
`insert_null_biases` method still exists in backend_entity.
And why it's temporary if it's not affect for FBC?

Reply:
We wonder to check out the algorithm's behavior with the null biases and unlimited threshold in the future.

Reply:
I am for clean code, if the function is not used then there is no point in keeping it in the code base. It can always be brought back from history.

Reply:
No problem, dropped it.